package.json

```diff
@@ -8,7 +8,7 @@
     "scripts": {
         "build": "webpack --env prod --env clean",
         "build:dev": "webpack --env clean",
-        "start": "concurrently \"npm:start:server\" \"npm:start:client\"",
+        "start": "start-server-and-test start:client 8080 start:server",
```

Comment:
🤔 i guess maybe it doesn't make a difference in this use case but this should be the other way around, first arg is server command and last is client

Reply:
flipped the arguments :)